File: 3630start.cpp
#include <iostream>
#include <cstdlib> // For std::atoi and std::atof

int main(int argc, char* argv[]) {
    // Check if the user provided the correct number of arguments
    if (argc != 4) {
        std::cerr << "Usage: " << argv[0] << " <int1> <int2> <double>\n";
        return 1; // Exit with error code
    }

    // Parse the command-line arguments
    int d = std::atoi(argv[1]);
    int N = std::atoi(argv[2]);
    double r = std::atof(argv[3]);
    
    // ******* add your code here 

    double volume = 0;
    double stdev = 0;
    double relerror=0.0;
    
    // *******

    // do not change the format below
    std::cout << "(r): " << r << std::endl;
    std::cout << "(d,N): " << d << " " << N << std::endl;
    std::cout << "volume: " << volume << std::endl;
    std::cout << "stat uncertainty: " << stdev << std::endl;
    std::cout << "relative error: " << relerror << std::endl;

    return 0; // Exit successfully
}

File: 3630start.py
import sys

def main():
    # Check if the user provided the correct number of arguments
    if len(sys.argv) != 4:
        print(f"Usage: {sys.argv[0]} <int1> <int2> <double>")
        sys.exit(1)  # Exit with error code

    # Parse the command-line arguments
    try:
        d = int(sys.argv[1])         # First integer
        N = int(sys.argv[2])         # Second integer
        r = float(sys.argv[3])       # First double

    except ValueError:
        print("Error: Please provide valid integers and doubles as arguments.")
        sys.exit(1)  # Exit with error code

    # ******* Add your code here

    volume = 0.0
    stdev = 0.0
    relerror=0.0

    # *******

    # Do not change the format below
    print(f"(r): {r}")
    print(f"(d,N): {d} {N}")
    print(f"volume: {volume}")
    print(f"stat uncertainty: {stdev}")
    print(f"relative error: {relerror}")

if __name__ == "__main__":
    main()
